### PR TITLE
New entry for docs repository "setUTCFullYear"

### PR DESCRIPTION
**Pull request by Beto-Garcia**
### Description

Addresses issue [#2548](https://github.com/Codecademy/docs/issues/2548)

### Type of Change

- Adding a new entry to the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md:19]:
```suggestion
```pseudo 
```

File: content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md:13]:
```suggestion
## Date.prototype.setUTCFullYear()
```

File: content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md:25]:
```suggestion
## Parameters
```

File: content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md:37]:
```suggestion
## Return value
```

File: content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md:42]:
```suggestion
## Description
```

File: content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md:49]:
```suggestion
## Examples
```

File: content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md:46]:
```suggestion
If the `monthValue` and `dateValue` parameters are not specified, the values returned from the *`getUTCMonth()`* and *`getUTCDate()`* methods are used.

If a parameter specified is outside of the expected range, `setUTCFullYear()` attempts to update the other parameters and the date information in the *`Date`* object accordingly. For example, if 15 is specified for `monthValue`, the year is incremented by 1 (`yearValue + 1`), and 3 is used for the month.
```
We don't use pronouns in docs contributions

File: content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md:53]:
```suggestion
```shell
```

File: content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md:51]:
```suggestion
Using `setUTCFullYear()`:
```

File: content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md:3]:
```suggestion
Description: 'Used to set the year value of a Date object in UTC (Coordinated Universal Time)'
```
Description should be small and descriptive explanation of what the function will does.

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Some changes from my side @Beto-Garcia :)

**Comment by SSwiniarski** [comment]
@Beto-Garcia tag me once you address @dakshdeepHERE's comments and I will do a first maintainer review.

File: content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md:10]:
```suggestion
Tags:
  - 'Date'
  - 'Methods'
CatalogContent:
  - 'introduction-to-javascript'
  - 'paths/front-end-engineer-career-path'
```

File: content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md:14]:
```suggestion
```
No leading header, the title in the metadata provides that.

File: content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md:15]:
```suggestion
The **`setUTCFullYear()`** method of `Date` instances changes the year for the date according to UTC (Coordinated Universal Time).
```

File: content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md:3]:
```suggestion
Description: 'Used to set the year value of a Date object in UTC (Coordinated Universal Time).'
```

File: content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md:25]:
```suggestion
Parameters:
```

File: content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md:34]:
```suggestion
- `yearValue`: An integer representing the year. For example, 1995.
- `monthValue`: (Optional) An integer representing the month: 0 for January, 1 for February, and so on.
- `dateValue`: (Optional) An integer between 1 and 31 representing the day of the month. If `dateValue` is specified, `monthValue` must also be specified.
```
No first- or second-person pronouns in Docs.

File: content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md:46]:
```suggestion
If the `monthValue` and `dateValue` parameters are not specified, the values returned from the `getUTCMonth()` and `getUTCDate()` methods are used.

> **Note:** If a parameter specified is outside of the expected range, `setUTCFullYear()` attempts to update the other parameters and the date information in the `Date` object accordingly. For example, if 15 is specified for `monthValue`, the year is incremented by 1 (`yearValue + 1`), and 3 is used for the month.
```

File: content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCFullYear/setUTCFullYear.md:56]:
````suggestion

## Example

Using `setUTCFullYear()`:

```shell
const theBigDay = new Date('2023-03-01');
theBigDay.setUTCFullYear(1997);
console.log(theBigDay);
```

This code will output the following:

```shell
1997-03-01T00:00:00.000Z
```
````

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Beto-Garcia I committed all of @dakshdeepHERE's comments to get this moving. I did a first maintainer review and left a few comments.

**Review by SSwiniarski — APPROVED** [review]
@Beto-Garcia In the interest of getting this moving, I implemented all my suggestions so this is ready for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@Beto-Garcia @SSwiniarski Just made a few formatting edits. Looks good to merge

**Pull request merged**
